integration_tests/tests/dashboard.cy.ts

```diff
@@ -28,17 +28,6 @@ context('Dashboard', () => {
     dashboardPage.shouldNotShowCard('manage')
   })
 
```

Comment:
@edavey it would be good to hear your feedback on this. We didn't think that a user without roles should be able to access 'Apply'.

Reply:
We agreed on a call with @edavey  that all users including those without roles should be able to access 'Apply' as having a Delius account (and therefore access to the service) is sufficient.